test/functional/mining_prioritisetransaction.py

```diff
@@ -116,7 +116,7 @@ def run_test(self):
         tx_id = self.nodes[0].decoderawtransaction(tx_hex)["txid"]
 
         # This will raise an exception due to min relay fee not being met
-        assert_raises_rpc_error(-26, "min relay fee not met (code 66)", self.nodes[0].sendrawtransaction, tx_hex)
+        assert_raises_rpc_error(-26, "min relay fee not met, 0 < 134 (code 66)", self.nodes[0].sendrawtransaction, tx_hex)
```

Comment:
```suggestion
        assert_raises_rpc_error(-26, "min relay fee not met, 0 < 191 (code 66)", self.nodes[0].sendrawtransaction, tx_hex)
```

`mining_prioritisetransaction.py` fails here obviously because of different tx size